webapp/lib/controller.dart

```diff
@@ -19,12 +19,23 @@ enum UIAction {
   systemEventsDataUpdated
 }
 
+enum ChartPeriodFilters {
+  days1,
+  days8,
+  days15,
+  month1,
```

Comment:
can we also have an `allTime` option? Or perhaps the default option could be used to show all of the data?

Reply:
Add new option `allTime`. It will show all data